webapp/src/main/java/com/box/l10n/mojito/service/repository/statistics/RepositoryLocaleStatisticCalculation.java

```diff
@@ -0,0 +1,104 @@
+package com.box.l10n.mojito.service.repository.statistics;
+
+public class RepositoryLocaleStatisticCalculation {
+
+  private long translatedCount = 0;
+
+  private long translatedWordCount = 0;
+
+  private long translationNeededCount = 0;
```

Comment:
suggestion: use AtomicLong so that it can support parallelisation of the stream?